src/components/organisms/FileTreePane/TreeItem.tsx

```diff
@@ -124,6 +126,19 @@ const TreeItem: React.FC<TreeItemProps> = props => {
     return style;
   }, [isMatchItem]);
 
+  React.useEffect(() => {
+    let mounted = true;
+    (async () => {
+      const res = await remotePath;
```

Comment:
Maybe we could just use the useEffect directly for setting the `remotePathUrl`, without the need for the useMemo. Or is there any particular reason for using both?

Reply:
Hey mate thought of this too but we are making a concat there when user clicks on open on github, thought maybe we dont want to do that unless the link is github and if user doesnt click.